apps/web-shop-front/src/graphql/product/useGetHomepageProducts.ts

```diff
@@ -0,0 +1,23 @@
+import { gql, useQuery } from '@apollo/client';
+import { Product } from '@team21/types'
+
+const query = gql`
+    query UseGetHomepageProducts($limit: Int!) {
+        products(order_by: {price: desc}, limit: $limit) {
+          id
+          image
+          name
+          price
+          description
+          quantity
+          category_id
+        }
+    }
+`;
+
+type UseGetHomepageProducts = {
+  products: Product[];
+};
```

Comment:
everything is great :D just a tip, you can omit unnecessary fields in query
also you cam use Omit utility type